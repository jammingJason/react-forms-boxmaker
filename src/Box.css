.Box-btn{
    border-color: white;
    background-color: red;
    color: white;
    text-align: right;
    justify-content: end;
    align-items: end
}
.Box-div{
    box-shadow: 3px 3px 3px;
}
