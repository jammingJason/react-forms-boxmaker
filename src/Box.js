import React from 'react';
import './Box.css';

const Box = ({ height, width, color, id }) => {
  // alert(height);

  const removeItem = (idx) => {
    const item = document.getElementById(idx);
    item.remove();
  };
  return (
    <div
      className="App Box-div"
      id={id}
      style={{
        textAlign: 'right',
        background: color,
        position: 'absolute',
        left: '45%',
        top: 250,
        height: height,
        width: width,
      }}
    >
      <button
        className="Box-btn"
        onClick={() => {
          removeItem(id);
        }}
      >
        X
      </button>
    </div>
  );
};

export default Box;
