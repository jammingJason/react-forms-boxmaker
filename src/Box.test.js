import { render, screen } from '@testing-library/react';
import Box from './Box';
test('renders Box', () => {
  render(<Box />);
  //   const linkElement = screen.getByText(/learn react/i);
  //   expect(linkElement).toBeInTheDocument();
});

test('matches snapshot', () => {
  const { asFragment } = render(<Box />);
  expect(asFragment()).toMatchSnapshot();
});
