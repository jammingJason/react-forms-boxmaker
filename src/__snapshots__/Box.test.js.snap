// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="App Box-div"
    style="text-align: right; position: absolute; left: 45%; top: 250px;"
  >
    <button
      class="Box-btn"
    >
      X
    </button>
  </div>
</DocumentFragment>
`;
